app/Renderers/Layouts/Campaign/CampaignExport.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Renderers\Layouts\Campaign;
+
+use App\Renderers\Layouts\Layout;
+use Illuminate\Support\Facades\Storage;
+
+class CampaignExport extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'created_by' => [
+                'key' => 'user.name',
+                'label' => 'campaigns.exports.created_by',
+                'render' => function ($model) {
+                    $html = '<a class="block break-all truncate" href="' . route('users.profile', [$model->user]) . '" target="_blank">' . $model->user->name . '</a>';
+                    return $html;
+                },
+            ],
+            'size' => [
+                'label' => 'campaigns.exports.size',
+                'render' => function ($model) {
+                    if ($model->size == 0) {
+                        return '<1 MB';
+                    }
+                    return $model->size . ' MB';
+                }
+            ],
+            'type' => [
+                'key' => 'type',
+                'label' => 'campaigns.exports.type',
+                'render' => function ($model) {
+                    $key = 'entities';
+                    /** @var \App\Models\CampaignExport $model */
+                    if ($model->type == \App\Models\CampaignExport::TYPE_ASSETS) {
+                        $key = 'assets';
+                    }
+
+                    return __('campaigns.exports.type_' . $key);
+                },
+            ],
+            'status' => [
+                'key' => 'status',
+                'label' => 'campaigns.exports.status',
+                'render' => function ($model) {
+                    $key = 'running';
+                    /** @var \App\Models\CampaignExport $model */
+                    if ($model->status == \App\Models\CampaignExport::STATUS_FAILED) {
+                        $key = 'failed';
+                    } elseif ($model->status == \App\Models\CampaignExport::STATUS_SCHEDULED) {
+                        $key = 'scheduled';
+                    } elseif ($model->status == \App\Models\CampaignExport::STATUS_FINISHED) {
+                        $key = 'finished';
+                    }
+
+                    return __('campaigns.exports.status_' . $key);
+                },
+            ],
+
+            'created_at' => [
+                'key' => 'created_at',
+                'label' => 'campaigns.exports.created',
+                'render' => function ($model) {
+                    $html = '';
+
+                    if (!empty($model->created_at)) {
```

Comment:
Confused how a model would never have a created_at value. But then the code is faster to read 
```php
if (empty($model->created_at)) {
   return '';
   }
   // the code
   ```